README.md

```diff
@@ -21,7 +21,7 @@ $ npm install nft-did-resolver
 ### Usage
 
 ```typescript
-import NftResolver, { NftResolverConfig } from 'nft-did-resolver'
+import { getResolver, NftResolverConfig } from 'nft-did-resolver'
```

Comment:
Not an issue but maybe the fact `NftResolverConfig` is a type could be explicitly indicated to avoid possible confusion and issues?
```suggestion
import { getResolver } from 'nft-did-resolver'
import type { NftResolverConfig } from 'nft-did-resolver'
```